### PR TITLE
Add support for profile_*_color in API

### PR DESCRIPTION
**Pull request by Rudloff**
This patch adds additional values to the user object returned by the API, based on the theme options (if Frio is used).
It is only a subset of the values [from Twitter](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/user-object) since they don't all apply to Friendica.

(Twidere mainly uses `profile_link_color` to tweak its interface so you can easily distinguish your accounts if you have several accounts in the app.)

### CODE REVIEW

File: include/api.php [hunk 1]
```diff
@@ -779,6 +780,41 @@ function api_get_user(App $a, $contact_id = null)
 		'network' => $uinfo[0]['network'],
 	);
 
+	// If this is a local user and it uses Frio, we can get its color preferences.
+	if ($ret['self']) {
+		$r = dba::p(
```
Comment (annando) [inline include/api.php]: please use ```dba::select``` for this case.

File: include/api.php [hunk 2]
```diff
@@ -779,6 +780,37 @@ function api_get_user(App $a, $contact_id = null)
...
 
+	// If this is a local user and it uses Frio, we can get its color preferences.
+	if ($ret['self']) {
+		$theme_info = dba::select('user', ['theme'], ['uid' => $ret['uid']], ['limit' => 1]);
+		if ($theme_info['theme'] === 'frio') {
+			$schema = PConfig::get($ret['uid'], 'frio', 'schema');
+			if (($schema) && ($schema != '---')) {
+				if (file_exists('view/theme/frio/schema/'.$schema.'.php')) {
+					$schemefile = 'view/theme/frio/schema/'.$schema.'.php';
+					require_once($schemefile);
```
Comment (MrPetovan) [inline include/api.php]: Please remove parentheses from language constructs.

File: include/api.php [hunk 3]
```diff
@@ -779,6 +780,37 @@ function api_get_user(App $a, $contact_id = null)
 		'network' => $uinfo[0]['network'],
 	);
 
+	// If this is a local user and it uses Frio, we can get its color preferences.
+	if ($ret['self']) {
+		$theme_info = dba::select('user', ['theme'], ['uid' => $ret['uid']], ['limit' => 1]);
+		if ($theme_info['theme'] === 'frio') {
+			$schema = PConfig::get($ret['uid'], 'frio', 'schema');
+			if (($schema) && ($schema != '---')) {
```
Comment (MrPetovan) [inline include/api.php]: Please remove extra parentheses in this condition.

Comment (annando) [reply include/api.php]: I, personally, would like to keep the one around ```($schema != '---')``` but remove the one around ```($schema)```. Although not needed, it increases readability.

**Pull request merged**